.github/workflows/workflow.yml

```diff
@@ -129,8 +129,10 @@ jobs:
       contents: write # IMPORTANT: mandatory for making GitHub Releases
       id-token: write # IMPORTANT: mandatory for sigstore
 
+    env:
+      GH_TOKEN: ${{ github.token }}
```

Comment:
This may lead to leaking the secret to third-party actions, especially, if the job gets new steps in the future. I recommend scoping the secret access to steps that need it.

Reply:
Thanks for the very good point.

Fixed in #15